### PR TITLE
Read REVISION and add it to the <header> + Warning

### PR DESCRIPTION
**Pull request by bascht**
(refs #621)

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -256,7 +256,7 @@ en:
   users:
     edit:
       allow_ignore_view: Show ignored hashtags publicly
-    show: 
+    show:
```
Comment (moonglum) [inline config/locales/en.yml]: What happened here?

Comment (bitboxer) [inline config/locales/en.yml]: Remove all da evil whitespace!

File: config/application.rb [hunk 2]
```diff
@@ -52,5 +52,8 @@ class Application < Rails::Application
 
     # Our weeks in analytics should start on Monday
     Groupdate.week_start = :mon
+
+    # Let the app know what version we're running (REVISION is written by Capistrano on deploy)
+    config.x.current_revision = File.exist?("#{Rails.root}/REVISION") ? File.read("#{Rails.root}/REVISION").strip : "UNKNOWN"
```
Comment (moonglum) [inline config/application.rb]: What is `config.x`?

Comment (bascht) [inline config/application.rb]: I stole that `x` from [the Rails Guide](http://guides.rubyonrails.org/configuring.html#custom-configuration). Is there a better way to do it?

File: config/locales/en.yml (hunk 1 above)
Comment (bascht) [inline config/locales/en.yml]: Whoops. Editor Hook. :blush:

File: config/application.rb (hunk 2 above)
Comment (moonglum) [inline config/application.rb]: I don't think it is bad in any way, I just never saw that and wondered what it is :smile:

**Comment by moonglum** [comment]
I'll give my :shipit: Thanks for your work, @bascht :heart:

**Pull request merged**